internal/services/pubkey_service.go

```diff
@@ -9,9 +12,18 @@ import (
 	"github.com/go-chi/render"
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
-	"net/http"
 )
 
+var (
+	GetPubkeyDao = (&PubkeyStore{}).GetPubkeyDao
+)
+
+type PubkeyStore struct{}
+
+func (*PubkeyStore) GetPubkeyDao(ctx context.Context) (dao.PubkeyDao, error) {
+	return (dao.GetPubkeyDao(ctx, db.DB))
+}
```

Comment:
I wanted to stub out the database completely and this was the only way I figured.
I'm reimplementing this method `GetPubkeyDao` to return `PubkeyStoreMock`

The db.DB makes it harder to stub out db, that is why I cant use
`var GetPubkeyDao = dao.GetPubkeyDao` but I need to actually add this "ugly" wrapper